imageprocessor/tests.py

```diff
@@ -104,6 +104,15 @@ def test_tag_search_post_request_works(self):
         self.assertTrue(response.status_code == 200)
         self.assertTrue(response.context['search_result'].get("total_count") >= 1)
 
+    def test_post_same_image_with_different_name(self):
+        client = Client()
+        with open(TEST_IMAGES_DIR + "/image5.jpg", "rb") as file1, open(TEST_IMAGES_DIR + "/same_as_image5.jpg", "rb") as file2:
```

Comment:
You don't need two test images, you can use image5 twice. I think it'd be better to reuse image5 because it will keep us from taking up extra space on Heroku